### PR TITLE
data-uploads/refresh-docs

### PR DESCRIPTION
**Pull request by eugsokolov**

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -293,3 +293,45 @@ prettyprints the single dimension dict:
             "section_pk": 707,
         },
     }
+
+How to Data Upload
+^^^^^^^^^^^^^^^^^^^
+
+Currently the API supports the ability to search for data groups one owns, check the processing status, and upload valid DataFrames. One must still upload data through the UI - specifying the schema. The API is meant to programmatically refresh the data.
```
Comment (reyna72) [inline docs/examples.rst]: maybe start something with "The API is meant to programmatically refresh existing upload. The initial upload that specifies the schema must still be uploaded via UI."

File: docs/examples.rst [hunk 2]
```diff
@@ -293,3 +293,45 @@ prettyprints the single dimension dict:
...
+
+..  code::
+
+    for data_group in dm.get_data_groups():
+        print(data_group)
+
+Alternatively, one can fetch a data group by its ID and view it's status:
```
Comment (reyna72) [inline docs/examples.rst]: "its" status

File: docs/examples.rst [hunk 3]
```diff
@@ -293,3 +293,45 @@ prettyprints the single dimension dict:
...
+
+..  code::
+
+    dg = dm.get_data_group_by_id(1012)
+    dg.get_current_status()
+
+To view the columns of the data group, and hence the schema, to verify the type of data we wish to reupload:
```
Comment (reyna72) [inline docs/examples.rst]: maybe rephrase?
To view the columns/schema of the data group in order to verify the type of data we wish to re-upload:

File: docs/examples.rst [hunk 4]
```diff
@@ -293,3 +293,45 @@ prettyprints the single dimension dict:
...
+        'end date': ['2019-01-02'],
+        'dummy data 1': [1],
+        'dummy data_2': [1],
+        'Ticker': ['AAP'],
+        ...
+    })
+    dg.start_data_refresh(df)
+    dg.get_current_status()
+
+One will notice the `status` of the data group object change.
```
Comment (reyna72) [inline docs/examples.rst]: maybe this?
The `status` of the data group object will change to reflect the latest status

Comment (eugsokolov) [reply docs/examples.rst]: thx

File: docs/examples.rst (hunk 3 above)
Comment (eugsokolov) [reply docs/examples.rst]: thx

File: docs/examples.rst (hunk 1 above)
Comment (eugsokolov) [reply docs/examples.rst]: thx

File: docs/examples.rst (hunk 4 above)
Comment (reyna72) [reply docs/examples.rst]: sorry this actually sounds like the `data_group.status` will change without calling `dg.get_current_status()` again. but that's not the case right?

Comment (eugsokolov) [reply docs/examples.rst]: i think that is discussed in the docs of the class - hence why i was vague and didn't specify exactly what to call in code

Comment (reyna72) [reply docs/examples.rst]: i see. given the context it should be clear then. also the function is well named

**Comment by reyna72** [comment]
lgtm
